### PR TITLE
feat: update navbar to match new design

### PR DESCRIPTION
**Pull request by seve**
#6128 
image

### CODE REVIEW

**Review by hthomas-czi — COMMENTED** [review]
Looks good! I have a couple requests:

- Increase the `font-weight` for the tabs from `medium` to `semibold` (`600`) to better differentiate between the new section labels and the tabs.
- The overall height of the nav bar should increase from `48px` to `56px` to maintain some vertical padding while accounting for the new "Applications" and "Census" labels.
- Increase the logo size to `28px` to maintain the same proportionality with the increased nav bar height.

**Comment by hthomas-czi** [comment]
@seve A couple other general comments while I'm waiting for rdev to refresh:

- The main content div should have `box-sizing: content-box` and `padding: 0px 40px` in addition to the `margin: 80px auto`, so the content doesn't hit the edge when the viewport is smaller
- The `font-weight` for the Embedding and Mode buttons should be `medium` (`500`) not `semibold`

File: frontend/src/components/Header/components/Nav/index.tsx [hunk 1]
```diff
@@ -6,7 +6,13 @@ import { track } from "src/common/analytics";
...
+import {
+  LinkWrapper,
+  NavItemContainer,
+  NavSection,
+  NavSectionTitle,
+  Nav as NavWrapper,
+} from "./style";
 import { BetaChip } from "../../style";
 
 export const CENSUS_LINK = "https://chanzuckerberg.github.io/cellxgene-census/";
```
Comment (kaloster) [inline frontend/src/components/Header/components/Nav/index.tsx:18]: can we please move this into a constants file?

File: frontend/src/components/Header/components/Nav/index.tsx [hunk 2]
```diff
@@ -17,7 +23,8 @@ interface Props {
 }
```
Comment (kaloster) [inline frontend/src/components/Header/components/Nav/index.tsx:24]: and this to a types file? thank you!

**Review by kaloster — COMMENTED** [review]
Solid work! Just a couple of comments. Thanks!

**Review by kaloster — APPROVED** [review]
LGTM! 🚀

**Pull request merged**